components/instructions/programs/streamflow.tsx

```diff
@@ -170,60 +180,63 @@ async function getStreamCreateDataUI(
 
     return (
       <div>
-        <div>
-          <span>Recipient:</span>
-          <span className="ml-2">{recipient.toBase58()}</span>
-        </div>
-
-        <span className="text-orange mt-2">
-          RECIPIENT SHOULD BE THE WALLET, NOT THE ASSOCIATED TOKEN ACCOUNT
-        </span>
-
-        <div className="mt-4">
-          <span>Start:</span>
-          {start == 0 && 'On approval'}
-          {start > 0 && (
-            <span className="ml-2">
-              {new Date(start * 1_000 /* second to ms */).toISOString()} UTC
-            </span>
-          )}
-        </div>
-
-        <div>
-          <span>Amount:</span>
-          <span className="ml-2">
-            {amountDeposited} {mintMetadata.symbol}
-          </span>
-        </div>
+        <InstructionDataUI>
+          <DataUIRow>
+            <DataUILabel label="Recipient" />
+            <DataUIAddress address={recipient} />
+          </DataUIRow>
+          <DataUIRow>
+            <DataUIWarning
+              className="text-orange"
```

Comment:
Should probably be orange by default if it's a warning

Or just use DataUIText with custom class. Warning is rather specific, not sure if it will be used in an other case